randovania/gui/ui_files/main_window.ui

```diff
@@ -644,6 +649,11 @@ For more information, check [Randovania Help](help://tab_multiworld)!</string>
     <string>Automatic Data Collection and Reporting</string>
    </property>
   </action>
+  <action name="menu_action_generate_another_process">
+   <property name="text">
+    <string>Generate games in another process</string>
+   </property>
+  </action>
```

Comment:
where's this handled?

Reply:
Thanks Qt Creator, I deleted that field. Not changing the PR because that's what I'm working on for another PR.